docs/content-services/search-filter.component.md

```diff
@@ -277,7 +280,7 @@ the next block of results.
 
 | Name | Type | Default | Description |
 | ---- | ---- | ------- | ----------- |
-| field | string |  | Specifies the facet field. |
+| field | string |  | Specifies the facet field. If the field string contains spaces, it must be escaped by wrapping it inside quotes - in order to get the correct response from the server.|
```

Comment:
Can we do this in the code when we build query? checking for spaces and wrapping the code so that developers don't need to remember this thing?